<html>
<head>
  <title>VEP summary</title>
  <script type="text/javascript" src="http://www.google.com/jsapi"></script>
  <script type="text/javascript">
    google.load('visualization', '1', {packages: ['corechart','table']});
  </script>
  <script type="text/javascript">
    
    function init() {
      // charts
      var var_class_pie = drawPie('var_class_pie', 'Variant classes', google.visualization.arrayToDataTable([['Variant class','Count'],['deletion',33],['insertion',37],['indel',223],['SNV',5522]]), null);
var var_class_table = drawTable('var_class_table', 'Variant classes', google.visualization.arrayToDataTable([['Variant class','Count'],['deletion',33],['insertion',37],['indel',223],['SNV',5522]]));

          google.visualization.events.addListener(var_class_pie, 'select', function() {
            var_class_table.setSelection(var_class_pie.getSelection());
          });
          google.visualization.events.addListener(var_class_table, 'select', function() {
            var_class_pie.setSelection(var_class_table.getSelection());
          });
        var var_cons_pie = drawPie('var_cons_pie', 'Consequences (most severe)', google.visualization.arrayToDataTable([['Consequence type','Count'],['splice_donor_variant',1],['stop_gained',24],['frameshift_variant',65],['stop_lost',2],['start_lost',3],['inframe_insertion',7],['inframe_deletion',33],['missense_variant',1457],['splice_polypyrimidine_tract_variant',7],['splice_donor_region_variant',1],['synonymous_variant',2107],['stop_retained_variant',3],['non_coding_transcript_exon_variant',45],['intron_variant',73],['upstream_gene_variant',1953],['downstream_gene_variant',34]]), {slices: [{color: "#ff7f50"}, {color: "#ff0000"}, {color: "#ff69b4"}, {color: "#ff0000"}, {}, {color: "#ff69b4"}, {color: "#ff69b4"}, {color: "#ffd700"}, {}, {}, {color: "#76ee00"}, {color: "#76ee00"}, {}, {color: "#02599c"}, {color: "#a2b5cd"}, {color: "#a2b5cd"}]});
var var_cons_table = drawTable('var_cons_table', 'Consequences (most severe)', google.visualization.arrayToDataTable([['Consequence type','Count'],['splice_donor_variant',1],['stop_gained',24],['frameshift_variant',65],['stop_lost',2],['start_lost',3],['inframe_insertion',7],['inframe_deletion',33],['missense_variant',1457],['splice_polypyrimidine_tract_variant',7],['splice_donor_region_variant',1],['synonymous_variant',2107],['stop_retained_variant',3],['non_coding_transcript_exon_variant',45],['intron_variant',73],['upstream_gene_variant',1953],['downstream_gene_variant',34]]));

          google.visualization.events.addListener(var_cons_pie, 'select', function() {
            var_cons_table.setSelection(var_cons_pie.getSelection());
          });
          google.visualization.events.addListener(var_cons_table, 'select', function() {
            var_cons_pie.setSelection(var_cons_table.getSelection());
          });
        var consequences_pie = drawPie('consequences_pie', 'Consequences (all)', google.visualization.arrayToDataTable([['Consequence type','Count'],['splice_donor_variant',1],['stop_gained',24],['frameshift_variant',67],['stop_lost',3],['start_lost',3],['inframe_insertion',7],['inframe_deletion',34],['missense_variant',1482],['splice_polypyrimidine_tract_variant',7],['splice_region_variant',2],['splice_donor_region_variant',2],['stop_retained_variant',4],['synonymous_variant',2139],['non_coding_transcript_exon_variant',48],['intron_variant',134],['non_coding_transcript_variant',2],['upstream_gene_variant',16800],['downstream_gene_variant',17557]]), {slices: [{color: "#ff7f50"}, {color: "#ff0000"}, {color: "#ff69b4"}, {color: "#ff0000"}, {}, {color: "#ff69b4"}, {color: "#ff69b4"}, {color: "#ffd700"}, {}, {color: "#ff7f50"}, {}, {color: "#76ee00"}, {color: "#76ee00"}, {}, {color: "#02599c"}, {}, {color: "#a2b5cd"}, {color: "#a2b5cd"}]});
var consequences_table = drawTable('consequences_table', 'Consequences (all)', google.visualization.arrayToDataTable([['Consequence type','Count'],['splice_donor_variant',1],['stop_gained',24],['frameshift_variant',67],['stop_lost',3],['start_lost',3],['inframe_insertion',7],['inframe_deletion',34],['missense_variant',1482],['splice_polypyrimidine_tract_variant',7],['splice_region_variant',2],['splice_donor_region_variant',2],['stop_retained_variant',4],['synonymous_variant',2139],['non_coding_transcript_exon_variant',48],['intron_variant',134],['non_coding_transcript_variant',2],['upstream_gene_variant',16800],['downstream_gene_variant',17557]]));

          google.visualization.events.addListener(consequences_pie, 'select', function() {
            consequences_table.setSelection(consequences_pie.getSelection());
          });
          google.visualization.events.addListener(consequences_table, 'select', function() {
            consequences_pie.setSelection(consequences_table.getSelection());
          });
        var coding_pie = drawPie('coding_pie', 'Coding consequences', google.visualization.arrayToDataTable([['Consequence type','Count'],['stop_gained',24],['frameshift_variant',67],['stop_lost',3],['start_lost',3],['inframe_insertion',7],['inframe_deletion',34],['missense_variant',1482],['synonymous_variant',2139],['stop_retained_variant',4]]), {slices: [{color: "#ff0000"}, {color: "#ff69b4"}, {color: "#ff0000"}, {}, {color: "#ff69b4"}, {color: "#ff69b4"}, {color: "#ffd700"}, {color: "#76ee00"}, {color: "#76ee00"}]});
var coding_table = drawTable('coding_table', 'Coding consequences', google.visualization.arrayToDataTable([['Consequence type','Count'],['stop_gained',24],['frameshift_variant',67],['stop_lost',3],['start_lost',3],['inframe_insertion',7],['inframe_deletion',34],['missense_variant',1482],['synonymous_variant',2139],['stop_retained_variant',4]]));

          google.visualization.events.addListener(coding_pie, 'select', function() {
            coding_table.setSelection(coding_pie.getSelection());
          });
          google.visualization.events.addListener(coding_table, 'select', function() {
            coding_pie.setSelection(coding_table.getSelection());
          });
        var chr_bar = drawBar('chr_bar', 'Variants by chromosome', google.visualization.arrayToDataTable([['Chromosome','Count'],['I',71],['II',864],['III',325],['IV',244],['IX',168],['Mito',105],['V',46],['VI',92],['VII',936],['VIII',49],['X',504],['XI',62],['XII',1311],['XIII',441],['XIV',184],['XV',280],['XVI',133]]), {legend: {position: "none"}});
var chr_table = drawTable('chr_table', 'Variants by chromosome', google.visualization.arrayToDataTable([['Chromosome','Count'],['I',71],['II',864],['III',325],['IV',244],['IX',168],['Mito',105],['V',46],['VI',92],['VII',936],['VIII',49],['X',504],['XI',62],['XII',1311],['XIII',441],['XIV',184],['XV',280],['XVI',133]]));

          google.visualization.events.addListener(chr_bar, 'select', function() {
            chr_table.setSelection(chr_bar.getSelection());
          });
          google.visualization.events.addListener(chr_table, 'select', function() {
            chr_bar.setSelection(chr_table.getSelection());
          });
        var chr_I_area = drawArea('chr_I_area', 'Distribution of variants on chromosome I', google.visualization.arrayToDataTable([['Position (mb)','Count'],['0',71]]), {hAxis: {title: "Position (mb)", textStyle: {fontSize: 8}}, legend: {position: "none"}});
var chr_II_area = drawArea('chr_II_area', 'Distribution of variants on chromosome II', google.visualization.arrayToDataTable([['Position (mb)','Count'],['0',864]]), {hAxis: {title: "Position (mb)", textStyle: {fontSize: 8}}, legend: {position: "none"}});
var chr_III_area = drawArea('chr_III_area', 'Distribution of variants on chromosome III', google.visualization.arrayToDataTable([['Position (mb)','Count'],['0',325]]), {hAxis: {title: "Position (mb)", textStyle: {fontSize: 8}}, legend: {position: "none"}});
var chr_IV_area = drawArea('chr_IV_area', 'Distribution of variants on chromosome IV', google.visualization.arrayToDataTable([['Position (mb)','Count'],['0',181],['1',63]]), {hAxis: {title: "Position (mb)", textStyle: {fontSize: 8}}, legend: {position: "none"}});
var chr_IX_area = drawArea('chr_IX_area', 'Distribution of variants on chromosome IX', google.visualization.arrayToDataTable([['Position (mb)','Count'],['0',168]]), {hAxis: {title: "Position (mb)", textStyle: {fontSize: 8}}, legend: {position: "none"}});
var chr_Mito_area = drawArea('chr_Mito_area', 'Distribution of variants on chromosome Mito', google.visualization.arrayToDataTable([['Position (mb)','Count'],['0',105]]), {hAxis: {title: "Position (mb)", textStyle: {fontSize: 8}}, legend: {position: "none"}});
var chr_V_area = drawArea('chr_V_area', 'Distribution of variants on chromosome V', google.visualization.arrayToDataTable([['Position (mb)','Count'],['0',46]]), {hAxis: {title: "Position (mb)", textStyle: {fontSize: 8}}, legend: {position: "none"}});
var chr_VI_area = drawArea('chr_VI_area', 'Distribution of variants on chromosome VI', google.visualization.arrayToDataTable([['Position (mb)','Count'],['0',92]]), {hAxis: {title: "Position (mb)", textStyle: {fontSize: 8}}, legend: {position: "none"}});
var chr_VII_area = drawArea('chr_VII_area', 'Distribution of variants on chromosome VII', google.visualization.arrayToDataTable([['Position (mb)','Count'],['0',928],['1',8]]), {hAxis: {title: "Position (mb)", textStyle: {fontSize: 8}}, legend: {position: "none"}});
var chr_VIII_area = drawArea('chr_VIII_area', 'Distribution of variants on chromosome VIII', google.visualization.arrayToDataTable([['Position (mb)','Count'],['0',49]]), {hAxis: {title: "Position (mb)", textStyle: {fontSize: 8}}, legend: {position: "none"}});
var chr_X_area = drawArea('chr_X_area', 'Distribution of variants on chromosome X', google.visualization.arrayToDataTable([['Position (mb)','Count'],['0',504]]), {hAxis: {title: "Position (mb)", textStyle: {fontSize: 8}}, legend: {position: "none"}});
var chr_XI_area = drawArea('chr_XI_area', 'Distribution of variants on chromosome XI', google.visualization.arrayToDataTable([['Position (mb)','Count'],['0',62]]), {hAxis: {title: "Position (mb)", textStyle: {fontSize: 8}}, legend: {position: "none"}});
var chr_XII_area = drawArea('chr_XII_area', 'Distribution of variants on chromosome XII', google.visualization.arrayToDataTable([['Position (mb)','Count'],['0',1305],['1',6]]), {hAxis: {title: "Position (mb)", textStyle: {fontSize: 8}}, legend: {position: "none"}});
var chr_XIII_area = drawArea('chr_XIII_area', 'Distribution of variants on chromosome XIII', google.visualization.arrayToDataTable([['Position (mb)','Count'],['0',441]]), {hAxis: {title: "Position (mb)", textStyle: {fontSize: 8}}, legend: {position: "none"}});
var chr_XIV_area = drawArea('chr_XIV_area', 'Distribution of variants on chromosome XIV', google.visualization.arrayToDataTable([['Position (mb)','Count'],['0',184]]), {hAxis: {title: "Position (mb)", textStyle: {fontSize: 8}}, legend: {position: "none"}});
var chr_XV_area = drawArea('chr_XV_area', 'Distribution of variants on chromosome XV', google.visualization.arrayToDataTable([['Position (mb)','Count'],['0',274],['1',6]]), {hAxis: {title: "Position (mb)", textStyle: {fontSize: 8}}, legend: {position: "none"}});
var chr_XVI_area = drawArea('chr_XVI_area', 'Distribution of variants on chromosome XVI', google.visualization.arrayToDataTable([['Position (mb)','Count'],['0',133]]), {hAxis: {title: "Position (mb)", textStyle: {fontSize: 8}}, legend: {position: "none"}});
var protein_bar = drawBar('protein_bar', 'Position in protein', google.visualization.arrayToDataTable([['Position in protein (percentile)','Count'],['00-10%',358],['10-20%',374],['20-30%',388],['30-40%',401],['40-50%',386],['50-60%',343],['60-70%',399],['70-80%',388],['80-90%',371],['90-100%',353]]), {hAxis: {title: "Position in protein (percentile)", textStyle: {fontSize: 10}}, legend: {position: "none"}});

    }
    
    function drawPie(id, title, data, options) {    
      var pie = new google.visualization.PieChart(document.getElementById(id));
      pie.draw(data, options);
      return pie;
    }
    function drawBar(id, title, data, options) {
      var bar = new google.visualization.ColumnChart(document.getElementById(id));
      bar.draw(data, options);
      return bar;
    }
    function drawTable(id, title, data) {
      var table = new google.visualization.Table(document.getElementById(id));
      table.draw(data, null);
      return table;
    }
    function drawLine(id, title, data, options) {
      var line = new google.visualization.LineChart(document.getElementById(id));
      line.draw(data, options);
      return line;
    }
    function drawArea(id, title, data, options) {
      var area = new google.visualization.AreaChart(document.getElementById(id));
      area.draw(data, options);
      return area;
    }
    google.setOnLoadCallback(init);
  </script>
  
  
  <style type="text/css">
    body {
      font-family: arial, sans-serif;
      margin: 0px;
      padding: 0px;
    }
    
    a {color: #36b;}
    a.visited {color: #006;}
    
    .stats_table {
      margin: 5px;
    }
    
    td {
      padding: 5px;
    }

    th.gradient {
      height: auto;
    }

    .stats_table tr:nth-child(odd) {
      background-color: #f0f0f0;
    }
    
    h3 {
      color: #666;
    }
    
    .masthead {
      background-color: white;
      color: rgb(204, 221, 255);
      height: 80px;
      width: 100%;
      padding: 0px;
    }
    
    .main {
      padding: 10px;
    }
    
    .gradient {
      background: #333366; /* Old browsers */
      background: -moz-linear-gradient(left,  #333366 0%, #ffffff 100%); /* FF3.6+ */
      background: -webkit-gradient(linear, left top, right top, color-stop(0%,#333366), color-stop(100%,#ffffff)); /* Chrome,Safari4+ */
      background: -webkit-linear-gradient(left,  #333366 0%,#ffffff 100%); /* Chrome10+,Safari5.1+ */
      background: -o-linear-gradient(left,  #333366 0%,#ffffff 100%); /* Opera 11.10+ */
      background: -ms-linear-gradient(left,  #333366 0%,#ffffff 100%); /* IE10+ */
      background: linear-gradient(to right,  #333366 0%,#ffffff 100%); /* W3C */
      filter: progid:DXImageTransform.Microsoft.gradient( startColorstr='#333366', endColorstr='#ffffff',GradientType=1 ); /* IE6-9 */
      
      padding: 0px;
      height: 80px;
      width: 700px;
    }
    
    .main_content {
      margin-left: 300px;
    }
    
    .sidemenu {
      width: 260px;
      position: fixed;
      border-style: solid;
      border-width: 2px;
      border-color: rgb(51, 51, 102);
    }
    
    .sidemenu_head {
      width: 250px;
      background-color: rgb(51, 51, 102);
      color: rgb(204, 221, 255);
      padding: 5px;
    }
    
    .sidemenu_body {
      width: 250px;
      padding: 5px;
    }
  </style>
</head>
<body>
<div id="masthead" class="masthead">
  <div style="float: left; display: inline; padding: 10px; height: 80px;">
    <a href="http://www.ensembl.org/"><img src="http://static.ensembl.org/i/e-ensembl.png" width="185" height="45" alt="Ensembl"></a>
  </div>
  
  <div style="float: right; display: inline; height: 80px; background: white; padding: 10px;">
    <a href="http://www.ensembl.org/vep"><img src="http://www.ensembl.org/img/vep_logo.png" width="111" height="50" alt="VEP"></a>
  </div>
  <div class="gradient">
  </div>
</div>
<div class="main">
<div class="sidemenu"><div class="sidemenu_head">Links</div><div class="sidemenu_body"><ul><li><a href="#masthead">Top of page</a></li><li><a href="#run_stats">VEP run statistics</a></li><li><a href="#gen_stats">General statistics</a></li><li><a href="#var_class">Variant classes</a></li><li><a href="#var_cons">Consequences (most severe)</a></li><li><a href="#consequences">Consequences (all)</a></li><li><a href="#coding">Coding consequences</a></li><li><a href="#chr">Variants by chromosome</a></li><li><a href="#protein">Position in protein</a></li></ul></div></div><div class='main_content'><h3 id="run_stats">VEP run statistics</h3><table class="stats_table"><tr><td>VEP version (API)</td><td> 106 (106)</td></tr><tr><td>Annotation sources</td><td>saccharomyces_cerevisiae_core_53_106_4 on mysql-eg-publicsql.ebi.ac.uk</td></tr><tr><td>Species</td><td>saccharomyces_cerevisiae</td></tr><tr><td>Command line options</td><td><pre>--database --genomes --input_file /home/lukasz/Analisis/NGS_Analysis/strip_data/polymorf_4545.vcf --output_file /home/lukasz/Analisis/NGS_Analysis/strip_data/detection_4545.vep --species saccharomyces_cerevisiae</pre></td></tr><tr><td>Start time</td><td>2022-06-01 06:28:46</td></tr><tr><td>End time</td><td>2022-06-01 15:22:16</td></tr><tr><td>Run time</td><td>32010 seconds</td></tr><tr><td>Input file</td><td>/home/lukasz/Analisis/NGS_Analysis/strip_data/polymorf_4545.vcf</td></tr><tr><td>Output file</td><td>/home/lukasz/Analisis/NGS_Analysis/strip_data/detection_4545.vep</td></tr></table><h3 id="gen_stats">General statistics</h3><table class="stats_table"><tr><td>Lines of input read</td><td>5815</td></tr><tr><td>Variants processed</td><td>5815</td></tr><tr><td>Variants filtered out</td><td>0</td></tr><tr><td>Novel / existing variants</td><td>-</td></tr><tr><td>Overlapped genes</td><td>4712</td></tr><tr><td>Overlapped transcripts</td><td>4712</td></tr><tr><td>Overlapped regulatory features</td><td>-</td></tr></table><hr/><h3 id="var_class">Variant classes</h3><div id="var_class_pie" style="width: 800px; height: 200px">&nbsp;</div><div id="var_class_table" style="width: 800px; height: 200px">&nbsp;</div><hr/><h3 id="var_cons">Consequences (most severe)</h3><div id="var_cons_pie" style="width: 800px; height: 400px">&nbsp;</div><div id="var_cons_table" style="width: 800px; height: 200px">&nbsp;</div><hr/><h3 id="consequences">Consequences (all)</h3><div id="consequences_pie" style="width: 800px; height: 400px">&nbsp;</div><div id="consequences_table" style="width: 800px; height: 200px">&nbsp;</div><hr/><h3 id="coding">Coding consequences</h3><div id="coding_pie" style="width: 800px; height: 400px">&nbsp;</div><div id="coding_table" style="width: 800px; height: 200px">&nbsp;</div><hr/><h3 id="chr">Variants by chromosome</h3><div id="chr_bar" style="width: 800px; height: 200px">&nbsp;</div><div id="chr_table" style="width: 800px; height: 200px">&nbsp;</div><hr/><h3 id="chr_I">Distribution of variants on chromosome I</h3><div id="chr_I_area" style="width: 800px; height: 200px">&nbsp;</div><hr/><h3 id="chr_II">Distribution of variants on chromosome II</h3><div id="chr_II_area" style="width: 800px; height: 200px">&nbsp;</div><hr/><h3 id="chr_III">Distribution of variants on chromosome III</h3><div id="chr_III_area" style="width: 800px; height: 200px">&nbsp;</div><hr/><h3 id="chr_IV">Distribution of variants on chromosome IV</h3><div id="chr_IV_area" style="width: 800px; height: 200px">&nbsp;</div><hr/><h3 id="chr_IX">Distribution of variants on chromosome IX</h3><div id="chr_IX_area" style="width: 800px; height: 200px">&nbsp;</div><hr/><h3 id="chr_Mito">Distribution of variants on chromosome Mito</h3><div id="chr_Mito_area" style="width: 800px; height: 200px">&nbsp;</div><hr/><h3 id="chr_V">Distribution of variants on chromosome V</h3><div id="chr_V_area" style="width: 800px; height: 200px">&nbsp;</div><hr/><h3 id="chr_VI">Distribution of variants on chromosome VI</h3><div id="chr_VI_area" style="width: 800px; height: 200px">&nbsp;</div><hr/><h3 id="chr_VII">Distribution of variants on chromosome VII</h3><div id="chr_VII_area" style="width: 800px; height: 200px">&nbsp;</div><hr/><h3 id="chr_VIII">Distribution of variants on chromosome VIII</h3><div id="chr_VIII_area" style="width: 800px; height: 200px">&nbsp;</div><hr/><h3 id="chr_X">Distribution of variants on chromosome X</h3><div id="chr_X_area" style="width: 800px; height: 200px">&nbsp;</div><hr/><h3 id="chr_XI">Distribution of variants on chromosome XI</h3><div id="chr_XI_area" style="width: 800px; height: 200px">&nbsp;</div><hr/><h3 id="chr_XII">Distribution of variants on chromosome XII</h3><div id="chr_XII_area" style="width: 800px; height: 200px">&nbsp;</div><hr/><h3 id="chr_XIII">Distribution of variants on chromosome XIII</h3><div id="chr_XIII_area" style="width: 800px; height: 200px">&nbsp;</div><hr/><h3 id="chr_XIV">Distribution of variants on chromosome XIV</h3><div id="chr_XIV_area" style="width: 800px; height: 200px">&nbsp;</div><hr/><h3 id="chr_XV">Distribution of variants on chromosome XV</h3><div id="chr_XV_area" style="width: 800px; height: 200px">&nbsp;</div><hr/><h3 id="chr_XVI">Distribution of variants on chromosome XVI</h3><div id="chr_XVI_area" style="width: 800px; height: 200px">&nbsp;</div><hr/><h3 id="protein">Position in protein</h3><div id="protein_bar" style="width: 800px; height: 200px">&nbsp;</div></div>
</div></body>
</html>
